tests/x11/smt.pm

```diff
@@ -25,7 +25,8 @@ sub run {
     turn_off_gnome_screensaver;
     become_root;
     # remove apache2-example-pages to avoid warning message shown during smt test
-    assert_script_run 'rpm -qa apache2-example-pages && rpm -e apache2-example-pages';
+    # if return 104 means the pache2-example-pages not exist, continue the test
+    zypper_call("rm apache2-example-pages", exitcode => [0, 104]);
```

Comment:
missing import `zypper_call` in
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5849/files#diff-db2b1c0542125969db763f4f3ec58a44R20

Reply:
like this `use utils qw(turn_off_gnome_screensaver zypper_call);`

Reply:
Updated, please help review again.